crates/shrs_core/src/builtin/help.rs

```diff
@@ -65,6 +70,24 @@ impl BuiltinCmd for HelpBuiltin {
                 }
                 ctx.out.println("")?;
             },
+            Commands::Plugin => {
+                let pgn = cli.args.join(" ");
+                let mut found = false;
+                for meta in sh.plugin_metas.iter() {
+                    if meta.name == pgn {
+                        found = true;
+                        ctx.out.println("")?;
+                        ctx.out.print_buf(styled_buf!(meta.name.clone().green()))?;
+                        ctx.out.println("")?;
+                        ctx.out.println(meta.help.clone())?;
+                        break; 
+                    }
+                }
+                if !found {
+                    ctx.out.print_buf(styled_buf!("Please specify a valid plugin name: help plugin <plugin-name>").red())?;
```

Comment:
Let's make the error message include the plugin name that was not found, maybe something like:
```
'Random Plugin' was not found, please specify a valid plugin name.
```

Reply:
Added.